test/Miki.Tests/Services/GuildServiceTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Threading.Tasks;
+using Miki.Bot.Models;
+using Miki.Services;
+using Miki.Services.Transactions;
+using Xunit;
+
+namespace Miki.Tests.Services
+{
+    public class GuildServiceTests : BaseEntityTest<MikiDbContext>
```

Comment:
Not a review comment, just a conversation piece: Technically these are integration tests instead of unit tests right now. We should probably consider making a standard on differentiating between integration and unit tests.

Reply:
Yes, I need to be caught up with this. A full discussion on it would be appreciated!